src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/nfs-export.pipe.ts

```diff
@@ -0,0 +1,10 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'nfsExport'
+})
+export class NfsExportPipe implements PipeTransform {
+  transform(value: object): string {
+    return `${value['path']}${value['pseudo']}`.split('//').join('/');
```

Comment:
I don't understand what this transformation is doing, can you give some examples?

Reply:
@rjfd It's just a replacement of `//` by `/`, you can find some examples in the corresponding unit test: https://github.com/ceph/ceph/blob/68badbd25cc33b15f497bfcf8f8096e31fc30b4a/src/pybind/mgr/dashboard/frontend/src/app/shared/pipes/nfs-export.pipe.spec.ts

Reply:
In that case I don't understand what's the reason to do this transformation. We shouldn't be joining the the "path" with the "pseudo path".

IMO, we should show two columns, one with the path, and other with the "pseudo path", or show the two paths separated in a single column.

Reply:
@rjfd `Pseudo` is now displayed in a separate column:

![Screenshot from 2019-09-16 11-28-52](https://user-images.githubusercontent.com/14297426/64951439-d0f4fb00-d875-11e9-9cda-7233cd291c43.png)